backend/tests/unit/core/constraint_validators/test_node_inherit_from_update.py

```diff
@@ -0,0 +1,60 @@
+import uuid
+from itertools import chain
+
+from infrahub.core import registry
+from infrahub.core.branch import Branch
+from infrahub.core.constants import PathType, SchemaPathType
+from infrahub.core.initialization import create_branch
+from infrahub.core.path import DataPath, SchemaPath
+from infrahub.core.validators.model import SchemaConstraintValidatorRequest
+from infrahub.core.validators.node.inherit_from import NodeInheritFromChecker
+from infrahub.database import InfrahubDatabase
+
+
+async def test_add_generic_success(db: InfrahubDatabase, default_branch: Branch, animal_person_schema):
+    branch = await create_branch(branch_name="branch", db=db)
+
+    dog_schema = registry.schema.get_node_schema(name="TestDog")
+    dog_schema.inherit_from.append("TestXXX")
+    request = SchemaConstraintValidatorRequest(
+        branch=branch,
+        constraint_name="node.generate_profile.update",
```

Comment:
it doesn't really matter for this test, but it might be good to update the name of the constraint to `node.inherit_from.update` and you could add `assert checker.supports(request)` to validate it

Reply:
Good catch